demo_umami_content/src/InstallHelper.php

```diff
@@ -244,6 +245,40 @@ public function importPages() {
     return $this;
   }
 
+  /**
+   * Imports block contents.
+   *
+   * @return $this
+   */
+  public function importBlockContent() {
+    $block_content_metadata = [
+      'umami_recipes_banner' => [
+        'uuid' => '4c7d58a3-a45d-412d-9068-259c57e40541',
+        'info' => 'Umami Recipes Banner',
+        'type' => 'banner_block',
+        'field_title' => [
+          'value' => 'Baked Camembert with garlic, calvados and salami',
+        ],
+        'field_content_link' => [
+          'target_id' => 10,
+        ],
+        'field_summary' => [
+          'value' => 'Nullam id dolor id nibh ultricies vehicula ut id elit. Nullam id dolor id nibh ultricies vehicula ut id elit. Nullam id dolor id nibh ultricies vehicula ut id elit.',
+        ],
+        'field_banner_image' => [
+          'target_id' => $this->getImage(drupal_get_path('theme', 'umami') . '/' . 'images/jpg/placeholder--atharva-lele-210748-pshopped.jpg'),
+        ],
+      ],
+    ];
+
+    // Create block content.
+    foreach ($block_content_metadata as $block) {
+      $block_content = $this->entityTypeManager->getStorage('block_content')->create($block);
+      $block_content->save();
```

Comment:
We don't mark this block for removal.
Is that OK?
If we do remove it, we'll also have to remove the block placement.
```php
$this->storeCreatedContentUuids([$block_content->uuid() => 'block_content']);
$placement = $this->entityTypeManager->getStorage('block')->load('umamirecipesbanner');
$this->storeCreatedContentUuids([$placement->uuid() => 'block']);
```
Would also require us to move the block placement (the block.block.umamirecipesbanner.yml file) into this module instead of in the profile, so reinstalling re-adds it.

Reply:
We will have to decide on how to remove every block we place via configs on uninstall.